src/shmem_internal.h

```diff
@@ -303,6 +303,20 @@ extern unsigned int shmem_internal_rand_seed;
         }                                                                               \
     } while (0)
 
+#define SHMEM_ERR_CHECK_SIG_OP(op)                                                      \
+    do {                                                                                \
+        switch(op) {                                                                    \
+            case SHMEM_SIGNAL_SET:                                                      \
+            case SHMEM_SIGNAL_ADD:                                                      \
+                break;                                                                  \
+            default:                                                                    \
+                fprintf(stderr, "ERROR: %s(): Argument \"%s\", "                        \
+                                "invalid atomic operation for signal (%d)\n",           \
+                        __func__, #op, (int) (op));                                     \
```

Comment:
Oh, how I wish these checks used `RAISE_ERROR_MSG`...

Reply:
Do you want me to change this here? Or should we do it in a new PR changing all the instances?

Reply:
No need for a change on this PR. There's nothing broken here, I am just being OCD. 👍